src/lib/types/AgentLiveSchema.ts

```diff
@@ -76,119 +59,129 @@ type SpeakModel =
+  /**
+   * Set to true to enable experimental features.
+   * @default false
+   */
+  experimental?: boolean;
   audio: {
     input?: {
       /**
-       * @default 1
+       * @default "linear16"
        */
-      channels?: number;
       encoding: AudioEncoding;
       /**
-       * @default false
+       * @default 16000
        */
-      multichannel?: boolean;
-      sampleRate: number;
+      sample_rate: number;
     };
     /**
      * @see https://developers.deepgram.com/docs/tts-media-output-settings#audio-format-combinations
      */
-    output?: AudioFormat;
-  };
-  agent: {
-    listen: {
+    output?: {
+      encoding?: string;
+      sample_rate?: number;
+      bitrate?: number;
       /**
-       * @see https://developers.deepgram.com/docs/model
+       * @default "none"
        */
-      model: ListenModel;
-      /**
-       * @see https://developers.deepgram.com/docs/keyterm
-       */
-      keyterms?: string[];
+      container?: string;
     };
-    speak: {
+  };
+  agent: {
+    language?: {
       /**
-       * @see https://developers.deepgram.com/docs/tts-models
+       * ISO 639-1 language code for agent language.
+       * @default "en"
        */
-      model: SpeakModel;
+      type: string;
+    };
+    listen?: {
+      provider: {
+        type: "deepgram";
+        /**
+         * @see https://developers.deepgram.com/docs/model
+         */
+        model: ListenModel;
+        /**
+         * Only available for Nova 3.
+         * @see https://developers.deepgram.com/docs/keyterm
+         */
+        keyterms?: string[];
+      };
+    };
+    speak?: {
+      provider: {
+        type: "deepgram" | "eleven_labs" | "cartesia" | "open_ai" | string;
+        /**
+         * Deepgram OR OpenAI model to use.
+         */
+        model?: SpeakModel;
+        /**
+         * Eleven Labs OR Cartesia model to use.
+         */
+        model_id?: string;
+        /**
+         * Cartesia voice configuration.
+         */
+        voice?: {
+          mode: string;
+          id: string;
+        };
+        /**
+         * Optional Cartesia language.
+         */
+        language?: string;
+        /**
+         * Optional Eleven Labs voice.
+         */
+        language_code?: string;
+      };
+      endpoint?: {
+        url?: string;
+        headers?: Record<string, string>;
+      };
     };
     /**
      * @see https://developers.deepgram.com/reference/voicebot-api-phase-preview#supported-llm-providers-and-models
      */
-    think: ThinkModel;
-  };
-  context?: {
-    /**
-     * LLM message history (e.g. to restore existing conversation if websocket disconnects)
-     */
-    messages: { role: "user" | "assistant"; content: string }[];
-    /**
-     * Whether to replay the last message, if it is an assistant message.
-     */
-    replay: boolean;
+    think?: {
+      provider: {
+        type: "deepgram" | "open_ai" | "anthropic" | "x_ai" | "amazon_bedrock" | string;
```

Comment:
```suggestion
        type: "deepgram" | "open_ai" | "anthropic" | "x_ai" | string;
```

Reply:
@naomi-lgbt  bedrock was confirmed of not being supported today. I've since updated the API Spec.

Reply:
Removed!